### PR TITLE
Upgrade streamly version

### PR DESCRIPTION
**Pull request by rnjtranjan**

### CODE REVIEW

**Review by harendra-kumar — CHANGES_REQUESTED** [review]
Same changes needed as in streamly-examples.

File: src/Streamly/Coreutils/Ls.hs [hunk 1]
```diff
@@ -48,8 +48,9 @@ listDir dir =
 ls :: (Ls -> Ls) -> String -> Stream IO (Either String String)
 ls f dir = do
     let opt = f defaultConfig
+        mapper = either Dir.readEitherPaths (const Stream.nil)
     case lsRecursive opt of
         Off -> listDir dir
         On ->
             let start = Stream.fromPure (Left ".")
-              in Stream.iterateMapLeftsWith Stream.ahead listDir start
+              in Stream.concatIterateWith Concur.ahead2 mapper start
```
Comment (harendra-kumar) [inline src/Streamly/Coreutils/Ls.hs:56]: use `parConcatIterate`.

File: src/Streamly/Coreutils/Uniq.hs [hunk 2]
```diff
@@ -12,11 +12,11 @@ module Streamly.Coreutils.Uniq
 import Control.Monad.Catch (MonadThrow)
 import Data.Char (isSpace, toLower)
 import Streamly.Data.Stream (Stream)
+import Streamly.Internal.Data.Stream (catRights)
```
Comment (harendra-kumar) [inline src/Streamly/Coreutils/Uniq.hs:15]: Why Internal? You can just remove this import and use Stream.catRights, it is available in the qualified import of Stream.

**Review by harendra-kumar — CHANGES_REQUESTED** [review]

**Review by rnjtranjan — COMMENTED** [review]
looks good.

**Pull request merged**